models/articles.models.js

```diff
@@ -39,8 +39,11 @@ exports.selectArticleById = (article_id) => {
   return db
     .query(
       `
-  SELECT * FROM articles
-  WHERE article_id = $1;`,
+      SELECT articles.article_id,title,topic,articles.author,articles.body,articles.created_at,
+      articles.votes,article_img_url, COUNT(comment_id) ::INT as comment_count 
+      FROM articles JOIN comments ON comments.article_id = articles.article_id
```

Comment:
This will need to be a left join otherwise if there are no comments for the article_id it just won't be included in the table